client/src/client.rs

```diff
@@ -99,7 +104,7 @@ fn handle_defaults<'a, 'b>(
     let required_num = args.len() - defaults.len();
 
     if let Some(i) = first_non_null_optional_idx {
-        &args[..i+1]
+        &args[..i + 1]
```

Comment:
Did `rustfmt` do that? It looks weird that the space is between operator that has precedence. Also, can't we just `[..=i]`? Is the rustc version requirement too low?

Reply:
Yeah somehow my Vim plugin `:RustFmt` doesnt format the same as `cargo fmt`. I'm still not sure how that came. These are some tiny nits from `cargo fmt`.